data/containers/autoyast_containers.xml.ep

```diff
@@ -129,6 +129,10 @@
               <copy_on_write config:type="boolean">true</copy_on_write>
               <path>boot/grub2/s390x-emu</path>
             </subvolume>
+            <subvolume config:type="map">
+              <copy_on_write config:type="boolean">true</copy_on_write>
```

Comment:
`/var` is explicitly set as nocow everywhere

Reply:
then shall b 'false' here?

Reply:
Yes

Reply:
Going to update both palaces from true to false